studio/pages/api/pg-meta/[ref]/tables.ts

```diff
@@ -29,38 +29,29 @@ async function handler(req: NextApiRequest, res: NextApiResponse) {
 const handleGetAll = async (req: NextApiRequest, res: NextApiResponse) => {
   const headers = constructHeaders(req.headers)
 
-  const onlyAuthSchema = req.query['schema'] === 'auth'
-  const onlyStorageSchema = req.query['schema'] === 'storage'
-
-  const includeAuthSchema = true // req.query['auth-schema'] === 'true'
-  const includeStorageSchema = true // req.query['storage-schema'] === 'true'
+  const query = Object.entries(req.query).reduce((query, entry) => {
+    const [key, value] = entry
+    if (Array.isArray(value)) {
+      for (const v of value) {
+        query.append(key, v)
+      }
+    } else if (value) {
+      query.set(key, value)
+    }
+    return query
+  }, new URLSearchParams())
+
+  let url = `${PG_META_URL}/tables`
+  if (Object.keys(req.query).length > 0) {
+    url += `?${query}`
+  }
```

Comment:
```suggestion
  if (query.size > 0) {
    url += `?${query}`
  }
```

Reply:
.size was added in node 19. I think studio is still on node18? Not entirely sure

Reply:
eh, yes, it is :( 
https://github.com/supabase/supabase/blob/0ab13a20afc6c484e8b2b2ede20e2ea881c1875b/package.json#L52